lalrpop/Cargo.toml

```diff
@@ -37,22 +37,19 @@ unicode-xid = { version = "0.2", default_features = false }
 # library, disable it in your project by setting default_features = false.
 pico-args = { version = "0.5", default_features = false, optional = true }
 
+lalrpop-util = { path = "../lalrpop-util", version = "0.19.10" }
 
 [dev-dependencies]
 rand = "0.8"
 
-[dependencies.lalrpop-util]
-path = "../lalrpop-util"
-version = "0.19.10" # LALRPOP
-
 [features]
-default=["lexer", "pico-args"]
-
-# Feature used when developing LALRPOP. Tells the build script to use an existing lalrpop binary to
-# generate LALRPOPs own parser instead of using the saved parser.
-test = []
-
+default=["lexer", "unicode", "pico-args"]
```

Comment:
Is it intentional to enable those by default? From other comments I got the impression that they should be disabled in 0.20 even though it's a breaking change.

Reply:
`lalrpop` is used during build so it's nor really an issue for it to have more features by default.

Reply:
Ok. I'm currently reading the changes on my phone, so I might be missing the big picture, literally and figuratively.